system/common.xml

```diff
@@ -350,16 +350,18 @@
 				</copy>
 			</then>
 		</if>
-		<condition property="isSVTTestRepo" value="true">
-			<isset property="env.isSVTTestRepo"/>
-		</condition>
-		<condition property="isSVTTestRepo" value="false">
-   		 	<not>
-				<isset property="env.isSVTTestRepo"/>
-			</not>
+		
+		<condition property="IS_SVT_TESTREPO" value="true" else="false">
+			<and>
+				<isset property="env.IS_SVT_TESTREPO"/>
+				<equals arg1="${env.IS_SVT_TESTREPO}" arg2="true"/>
+			</and>
 		</condition>
```

Comment:
Do we need this condition check? Can we do it at line 364? 
```
<equals arg1="${env.IS_SVT_TESTREPO}" arg2="true"/>
```

Reply:
Sure , I will try doing there.